components/win-users/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "habitat_win_users"
+version = "0.0.0"
+authors = ["Matt Wrock <matt@mattwrock.com>"]
+description = "Habitat library for win32 account api calls"
+build = "../build.rs"
```

Comment:
I totally hear you there. It took a looong time to eek out this bit of code here. I see alot of it in our future so I'm hoping it goes faster.